events/registry_test.go

```diff
@@ -0,0 +1,168 @@
+package events
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/centrifuge/go-substrate-rpc-client/v4/events/test"
+	"github.com/centrifuge/go-substrate-rpc-client/v4/types"
+	"github.com/centrifuge/go-substrate-rpc-client/v4/types/codec"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCreateEventRegistry(t *testing.T) {
```

Comment:
For the sake of diversity, I would add kusama, and a few parachains from the ecosystem to showcase the parser. We can pick up as well statically the metadata as you do with Polkadot on block n. WDYT?

Reply:
Yup, sounds good to me. I'll definitely add more but in a static way to avoid failures due to network connections and so on.

I also plan on having a test suite that is using live data but that would be after the 2nd milestone is complete.

Reply:
👍